Directory.Packages.props

```diff
@@ -0,0 +1,36 @@
+<Project>
+  <!--
+                                          ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+                                          █████ ▀██ ██ ▄▄▄█▄▄ ▄▄███ ▄▄ ████ ▄▄ ██
+                                          █▀▀██ █ █ ██ ▄▄▄███ █████ ▄▄ █▀▀█ ▀▄ ██
+                                          █▄▄██ ██▄ ██ ▀▀▀███ █████ ▀▀ █▄▄█ ▀▀ ██
+                                          ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+  -->
+
+  <ItemGroup>
+    <!--
+      .net sdk
+    -->
+    <PackageVersion Include="Microsoft.Extensions.Logging.Console"                                       Version="8.0.0" />
```

Comment:
I kind-of like using .* on minor versions. It makes it less work-intensive, to upgrade every time a 3rd party has a patch version upgrade.

Do you have other specific thoughts about this? Do you see any drawbacks of using `"8.0.*"` on these dependencies?